RecoLocalTracker/SiPixelRecHits/src/SiPixelGenError.cc

```diff
@@ -308,6 +319,11 @@ bool SiPixelGenError::pushfile(const SiPixelGenErrorDBObject& dbobject,
       
       // next, layout the 1-d/2-d structures needed to store GenError
       
+      
+      theCurrentTemp.cotbetaY = new float[theCurrentTemp.head.NTy];
+      theCurrentTemp.cotbetaX = new float[theCurrentTemp.head.NTyx];
+      theCurrentTemp.cotalphaX = new float[theCurrentTemp.head.NTxx];
```

Comment:
are these eventually deleted somewhere?

Reply:
Good catch.  I don't think they are.    For templates, the equivalent call is in the destructor, which calls destroy() method in the template objects which cleans arrays like these.  So we need the same thing, but in PixelCPEGeneric.    It's been a while since I looked into this but I'll try to fix this pronto.

Having said that, this is in the boost branch which is currently disabled.  But we'll need to enable it for Phase 2 (larger templates, spanning larger local angles)  so this has got to be fixed.